libxrdp/xrdp_caps.c

```diff
@@ -466,7 +466,7 @@ xrdp_caps_process_codecs(struct xrdp_rdp *self, struct stream *s, int len)
         }
         else if (g_memcmp(codec_guid, XR_CODEC_GUID_REMOTEFX, 16) == 0)
         {
-            g_writeln("xrdp_caps_process_codecs: rfx codec id %d prop len %d",
+            g_writeln("xrdp_caps_process_codecs: RemoteFX, codec id %d prop len %d",
```

Comment:
If you are improving this message, you can as well add a comma after the first `%d`